esm_runscripts/chunky_parts.py

```diff
@@ -98,6 +98,10 @@ def set_chunk_calendar(config):
     next_date = config["general"]["next_date"]
 
     number_of_chunks_done = this_chunk_number // number_of_models
+   
+    # LA: get right number_of_chunks_done for last year in chunk
+    if config["general"]["model_queue"][0] == "model1" and config["general"]["last_run_in_chunk"]:
+        number_of_chunks_done = number_of_chunks_done - 1
```

Comment:
Why do we need to substract 1 to the `number_of_chunks_done` only for `model1` and not for `model2`?

Reply:
It is because of this:
```

    number_of_models = config["general"]["number_of_ic_models"]
    this_chunk_number = int(config["general"]["next_chunk_number"]) - 1
    ...

    number_of_chunks_done = this_chunk_number // number_of_models
```

When you, for example, run AWI-ESM and PISM in 5 year chunks each. And you are within in first PISM chunk (which is the second chunk in total):
`number_of_models` = 2
`next_chunk_number` = 2 (for years 1 to 4) and 3 (for last year in this chunk)
then: 
`this_chunk_number` = 1 (for years 1 to 4) and 2 (for last year in chunk)
`number_of_chunks_done` = 0 (for years 1 to 4) and 1 (for last year in chunk, although it is still the first PISM chunk)

So I guess it would be necessary to it for every second model. It is a bit unlucky that it says: 
`if config["general"]["model_queue"][0] == "model1"` while it is actually applied to `model2`. Maybe one can also say::
`if config["general"]["model_queue"][1] == "model2"` 
I will test this.